tests/integration/annotation_import/conftest.py

```diff
@@ -615,14 +643,21 @@ def model_run_with_training_metadata(rand_gen, model):
 
 @pytest.fixture
 def model_run_with_model_run_data_rows(client, configured_project,
-                                       model_run_predictions, model_run):
+                                       model_run_predictions, model_run,
+                                       wait_for_label_processing):
     configured_project.enable_model_assisted_labeling()
 
     upload_task = LabelImport.create_from_objects(
         client, configured_project.uid, f"label-import-{uuid.uuid4()}",
         model_run_predictions)
     upload_task.wait_until_done()
```

Comment:
so this is not working?

Reply:
The fixture was added to check for DR processing to be finished